ubuntu-simple/DockerfileREADME.md

```diff
@@ -1,9 +1,11 @@
 # Getting started with Weave and Docker on Ubuntu - Dockerfile #
 
-For our getting started with Weave and Docker on Ubuntu guide we created a small docker image, `fintanr/weave-gs-simple-hw`.  We have included the Dockerfile we used for creating this Docker image in our repo. While this is a very simple example it demonstrates how easy it is to create Docker images.
+For this _Getting Started with Weave and Docker on Ubuntu Guide_ we created a small docker image, `fintanr/weave-gs-simple-hw` and also included the Dockerfile that was used to create this Docker image. 
+
+While this is a very simple example, it demonstrates how easy it is to create Docker images.
 
 ```bash
-MAINTAINER    fintan@weave.works
+MAINTAINER    ilya@weave.works
```

Comment:
I would just take this line out completely or specify `help@weave.works` instead.